settings.gradle.kts

```diff
@@ -1,50 +1,17 @@
-import org.gradle.api.internal.FeaturePreviews
-
-pluginManagement {
-    plugins {
-        id("org.jetbrains.kotlin.jvm") version "1.8.10"
-        id("com.github.johnrengelman.shadow") version "8.1.0"
-    }
-}
 plugins {
-    id("com.gradle.enterprise").version("3.12.3")
-    id("com.gradle.common-custom-user-data-gradle-plugin").version("1.8.2")
+    id("com.gradle.enterprise").version("3.13")
+    id("com.gradle.common-custom-user-data-gradle-plugin").version("1.10")
 }
 
-dependencyResolutionManagement {
-    repositories {
-        maven {
-            name = "Gradle public repository"
-            url = uri("https://repo.gradle.org/gradle/public")
-            content {
-                includeGroup("net.rubygrapefruit")
-                includeModule("classycle", "classycle")
-                includeModule("flot", "flot")
-                includeModule("org.gradle", "gradle-tooling-api")
```

Comment:
Do we not want to resolve the tooling API from Gradle's internal artifact server?

Reply:
I didn't debug, but it seems the `dependencyResolutionManagement` section was having no effect. All dependencies were already being resolved from `mavenCentral`. I think this is OK.
Before change: https://ge.gradle.org/s/63b23yvefr5lm/dependencies/repositories?repoToggled=W1swXSxbMV0sWzJdLFszXV0
After change: https://ge.gradle.org/s/thvqexo2mx7py/dependencies/repositories?repoToggled=W1swXV0